src/transformers/modeling_roberta.py

```diff
@@ -78,9 +102,29 @@ def forward(self, input_ids=None, token_type_ids=None, position_ids=None, inputs
             else:
                 position_ids = self.create_position_ids_from_inputs_embeds(inputs_embeds)
 
-        return super().forward(
-            input_ids, token_type_ids=token_type_ids, position_ids=position_ids, inputs_embeds=inputs_embeds
-        )
+        # Copied from transformers.modeling_bert.BertEmbeddings.forward
```

Comment:
The other way round, we can add any custom code at the beginning, *then* compare to the original code, by just putting the comment later in the function body and not at the beginning.

The only thing that is impossible to do is to change the middle of the copied code.